.github/workflows/ci-backwards-compatibility.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     if: github.repository == 'fedimint/fedimint'
     name: "Backwards-compatibility tests"
     runs-on: buildjet-8vcpu-ubuntu-2004
-    timeout-minutes: 60
+    timeout-minutes: 75
```

Comment:
I don't think we should degraded for backward compat, but only for the current normal version tests.

Reply:
I think that's a good tradeoff. Running backwards-compatibility with all peers online takes ~50 minutes. The extra 5 minutes is because I've added latency tests to backwards-compatibility in this PR.

https://github.com/fedimint/fedimint/actions/runs/7924984781/job/21637415304?pr=4342

cc: @justinmoon @maan2003 for visibility

Reply:
Hmm, 4/4 for latency is flakey in back-compat. I'll keep investigating